pkgs/applications/video/shotcut/default.nix

```diff
@@ -10,29 +9,29 @@
 , pkg-config
 , qtbase
 , qtmultimedia
-, qtx11extras
 , qtwebsockets
-, qtquickcontrols2
-, qtgraphicaleffects
-, qmake
+, cmake
 , qttools
+, wrapQtAppsHook
 , gitUpdater
+, stdenv
+, fftw
 }:
 
 assert lib.versionAtLeast mlt.version "6.24.0";
 
-mkDerivation rec {
+stdenv.mkDerivation rec {
```

Comment:
suggest using `finalAttrs` instead of `rec`, so that overriding doesn't require respecifying cflags
https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes

Reply:
(btw this would be a separate commit)